build.gradle.kts

```diff
@@ -17,6 +17,9 @@ repositories {
 	maven("https://hub.spigotmc.org/nexus/content/repositories/public/")
 	maven("https://repo.md-5.net/content/groups/public/")
 	maven("https://repo.papermc.io/repository/maven-public/")
+	maven(url = "https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+		name = "sonatype-oss-snapshots"
+	}
```

Comment:
```suggestion
maven("https://s01.oss.sonatype.org/content/repositories/snapshots/")

Reply:
Should not be needed anymore in the newest version due to kyori adventure-bom has been released as 2.17.0